stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -586,8 +596,9 @@
 ;; The tokens will unlock and be returned to the Stacker (tx-sender) automatically.
 (define-public (stack-stx (amount-ustx uint)
                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 32))))
+                          (signing-key (buff 33))
                           (start-burn-ht uint)
-                          (lock-period uint))
+                          (lock-period uint)))
```

Comment:
Typo -- extra parenthesis.

Please make sure that this contract instantiates.  The reason your tests are failing is because there are syntax errors in `pox-4`, which prevents it from instantiating.